inference-engine/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests/convolution_backprop_data.cpp

```diff
@@ -143,6 +143,27 @@ INSTANTIATE_TEST_CASE_P(smoke_ConvolutionBackpropData2D_AutoPadding_OutputPaddin
                                 ::testing::Values(CommonTestUtils::DEVICE_CPU)),
                         ConvolutionBackpropLayerTest::getTestCaseName);
 
+INSTANTIATE_TEST_CASE_P(smoke_ConvolutionBackpropData2D_RuntimeError, ConvolutionBackpropLayerTest,
```

Comment:
`RuntimeError` too abstract name. Maybe 'EndPadComputation' would be better.

Reply:
Done a7941e5e69ca3d8cb33e992b4d296ac9da6f030c